src/pages/profile/external-identities/FenceAccount.ts

```diff
@@ -3,21 +3,21 @@ import _ from 'lodash/fp';
 import * as qs from 'qs';
 import { Fragment, useState } from 'react';
 import { div, h, h3, span } from 'react-hyperscript-helpers';
-import { FrameworkServiceLink, UnlinkFenceAccount } from 'src/components/external-account-links';
 import { Ajax } from 'src/libs/ajax';
 import { withErrorReporting } from 'src/libs/error';
 import * as Nav from 'src/libs/nav';
 import { useOnMount, useStore } from 'src/libs/react-utils';
 import { authStore } from 'src/libs/state';
 import * as Utils from 'src/libs/utils';
+import { FrameworkServiceLink } from 'src/pages/profile/external-identities/FrameworkServiceLink';
 import { linkStyles as styles } from 'src/pages/profile/external-identities/LinkStyles';
+import { UnlinkFenceAccount } from 'src/pages/profile/external-identities/UnlinkFenceAccount';
 import { SpacedSpinner } from 'src/pages/profile/SpacedSpinner';
 
-export const FenceLink = ({ provider: { key, name, expiresAfter, short } }) => {
+export const FenceAccount = ({ provider: { key, name, expiresAfter, short } }) => {
```

Comment:
Its probably a good idea to create a FenceAccountProps similar to how you have `FrameworkServiceLinkProps`